tests/features/fixtures/mazerunner/build.gradle

```diff
@@ -26,7 +26,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.4.2'
-        classpath 'com.bugsnag:bugsnag-android-gradle-plugin:4.5.2'
+        classpath 'com.bugsnag:bugsnag-android-gradle-plugin:4+'
```

Comment:
This change would need to be `4.+` to pickup the dependency. Is there a benefit to bumping it to 4.7.3 instead though? It feels like it'd be good to keep a specific version of the plugin to avoid introducing an extra moving part to CI.

Reply:
Sounds good.  If there's an internal ticket to reactivate assertions around buildUUID this can be updated at the same time.